dracut/30ignition/afterburn-network-kargs.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Afterburn Initrd Setup Network Kernel Arguments
+Documentation=https://coreos.github.io/afterburn/usage/initrd-network-cmdline/
+
+# This service may produce additional kargs fragments,
+# which are then consumed by dracut-cmdline(8).
+DefaultDependencies=no
+Before=dracut-cmdline.service
+Before=ignition-fetch.service
+After=systemd-journald.socket
+
+OnFailure=emergency.target
+OnFailureJobMode=isolate
+
+[Service]
+Environment=AFTERBURN_OPT_PROVIDER=--cmdline
+Environment=AFTERBURN_NETWORK_KARGS_DEFAULT='ip=dhcp,dhcp6'
```

Comment:
It still looks like it would write the default value out to the file making the `ip=…` command line handling be triggered where it previously wasn't, or?
I think we can use an empty default meaning that the fragment file is empty and the dracut network setup would not be triggered (didn't check the details on how the fragment file is processed).